### PR TITLE
Create CCC-TAC-Minutes-2023-04-20.md

### PR DESCRIPTION
**Pull request by krtaylor**

### CODE REVIEW

File: TAC/Meetings/2023/2023-04-20/CCC-TAC-Minutes-2023-04-20.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+* DM reviewed the current state of the action items
+
+
+**2023 Goals**
+
+* DM addressed 2023 [plans](https://docs.google.com/document/d/1BLsI0hv9ybHl-FBNqHp6bJzy6ng8yKs__556bTqBswc/edit)
+* Project Progression - [PR #160](https://github.com/confidential-computing/governance/pull/160)
+* Cross Project coordination
+  * Lily Sturmann (LS) presented the coordination goals
+    * Cross project integration - broadening of the description, adding or 2 upstream projects being used by more than one project
```
Comment (lkatalin) [inline TAC/Meetings/2023/2023-04-20/CCC-TAC-Minutes-2023-04-20.md:99]:
```suggestion
    * Cross project integration - LS proposed broadening of the goal description to: adding module maintained by another CCC project, or 2 upstream projects being used or contributed to by more than one project
```

**Review by lkatalin — CHANGES_REQUESTED** [review]

**Review by dcmiddle — APPROVED** [review]

File: TAC/Meetings/2023/2023-04-20/CCC-TAC-Minutes-2023-04-20.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+**Time**: 7a-9a US Pacific Time
+
+* **Zoom**: [https://zoom.us/j/184384055](https://zoom.us/j/184384055) (pw: 43690)
+
+* **Calendar**: [https://calendar.confidentialcomputing.io](https://calendar.confidentialcomputing.io),
+[ical](https://calendar.google.com/calendar/ical/c\_c0pcihr7n2n1k3a38i32d9ag10%40group.calendar.google.com/public/basic.ics),
+[Google Calendar](https://calendar.google.com/calendar/u/0/r?cid=c\_c0pcihr7n2n1k3a38i32d9ag10@group.calendar.google.com)
+
+* **Recordings**: [YouTube TAC Playlist](https://www.youtube.com/playlist?list=PLmfkUJc39uMjaB\_I1dYW72I44kr9QzG\_B)
```
Comment (dthaler) [inline TAC/Meetings/2023/2023-04-20/CCC-TAC-Minutes-2023-04-20.md:19]: This link doesn't seem to work for me.  Is it just me?

File: TAC/Meetings/2023/2023-04-20/CCC-TAC-Minutes-2023-04-20.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+      * Needs further discussion - Dave Thaler (DT) proposed the following change to the definition "There exists at least 2 projects (CCC maintained or not) that have more than one CCC project consuming them."
+    * Need to figure out how this should be tracked, maybe have projects self-report, investigate common dependencies
+    * DT suggested that his proposed wording may not be sufficient
+    * The module integration dependency types were recorded in the goal document
+    * DT Suggested: During yearly project review ask projects about upstream contributions, i.e. have you influenced an upstream in a way that potentially benefits other CCC projects. 
+    * LS asked if the CCC needs a common project slack channel to discuss these sorts of things
+    * To improve communication, proposed cross project integration meetup or virtual event, possible at Confidential Computing Summit
+    * KT brought up the possibility of at CCS of setting up regular project maintainer meetings, Lily to follow up with the EUAC and Outreach chairs for event presence
```
Comment (dthaler) [inline TAC/Meetings/2023/2023-04-20/CCC-TAC-Minutes-2023-04-20.md:107]: Should "Lily to follow up..." be listed as an `**ACTION**`?

File: TAC/Meetings/2023/2023-04-20/CCC-TAC-Minutes-2023-04-20.md (hunk 2 above)
Comment (dthaler) [inline TAC/Meetings/2023/2023-04-20/CCC-TAC-Minutes-2023-04-20.md:19]:
```suggestion
* **Recordings**: [YouTube TAC Playlist](https://www.youtube.com/playlist?list=PLmfkUJc39uMjaB_I1dYW72I44kr9QzG_B)
```

**Review by dthaler — APPROVED** [review]

**Comment by yashkmankad** [comment]
Approved.

(While I wait for the CODEOWNERS issue to get resolved)

**Pull request merged**